src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -67,6 +67,11 @@ public async Task<IActionResult> Get()
         [Route("admin/host/extensions/{id}")]
         public async Task<IActionResult> Delete(string id)
         {
+            if (!FileUtility.IsValidFileName(id))
```

Comment:
Can do! But if we know what a pkg name / id looks like, and we know it will always be in that format, we can implement a more specific check instead. If its more generic I will keep the file name check